libsolidity/interface/CompilerStack.cpp

```diff
@@ -325,6 +325,7 @@ bool CompilerStack::compile()
 
 void CompilerStack::link()
 {
+	solAssert(m_stackState >= AnalysisSuccessful, "");
```

Comment:
I guess this should be CompilationSuccessful.

Reply:
True, only based that on what is a required field. It should work after analysis, though it wouldn't contain anything.

Reply:
Added.